docs/topics/user_permissions.md

```diff
@@ -0,0 +1,51 @@
+# Users and permissions
+
+
+## What user permissions are defined in IoTP?
+
+
+In IoTP there are defined Services and SubServices. SubServices exists into Services.
+Tipicaly a Service represents a smartcity and all of the SubServices represents the verticals of that smartcity.
```

Comment:
A link to the multitenancy document (doc/multitenancy.md or something like that) could be included in this part (and revie that the content/terms is not redundant and keeps coherence between the two .md files).